Microsoft.Toolkit.Sample.Wpf.App/Program.cs

```diff
@@ -62,7 +62,7 @@ enum StartupKind
         [STAThread]
         public static void Main()
         {
-            var startupKind = StartupKind.Normal;
```

Comment:
Was this the part you said wasn't supported anymore?  Should we have a comment here calling that this must be set this way and throw an exception in the else case below?

Reply:
We should remove the else clause. It doesn't work. There will be a future PR with a more complex scenario covered.